### PR TITLE
DT-371 Create new AWS Backup vault for system drives

### PR DESCRIPTION
**Pull request by BenRamchandani**
Create a new AWS backup vault that targets all volumes with the "system-drive-backup" tag set to the environment (e.g. "test"). That should make it easy to add Delta and Orbeon. Backups are once a week and kept for 14 days.

Working for Japsersoft on test: <https://eu-west-1.console.aws.amazon.com/backup/home?region=eu-west-1#/backupvaults/details/system-drives-backup-vault-test>

### CODE REVIEW

File: terraform/modules/ebs_backup/variables.tf
Comment (PhoebeJackson1) [inline terraform/modules/ebs_backup/variables.tf:11]: I've not seen ? in these before, does this just mean irrelevant because it's defined by "FRI"?

File: terraform/modules/marklogic/marklogic_cf_template.yml
Comment (PhoebeJackson1) [inline terraform/modules/marklogic/marklogic_cf_template.yml:770]: Dunno if it matters but since terraform is normally funny about line endings FYI

**Review by PhoebeJackson1 — APPROVED** [review]
Looks good to me but I don't have a huge amount of context on the backups setup currently so maybe worth waiting for another review? 🤷

File: terraform/modules/ebs_backup/variables.tf
Comment (BenRamchandani) [reply terraform/modules/ebs_backup/variables.tf:11]: AWS use slightly different cron expressions to everyone else with 6 fields instead of 5, you can either specify day-of-month (`?`, so not specified here) or day-of-week (`FRI`) https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-cron-expressions.html

File: terraform/modules/marklogic/marklogic_cf_template.yml
Comment (BenRamchandani) [reply terraform/modules/marklogic/marklogic_cf_template.yml:770]: In this case it's to stop Terraform trying to always change this file, at least for me

**Comment by BenRamchandani** [comment]
> Looks good to me but I don't have a huge amount of context on the backups setup currently so maybe worth waiting for another review? 🤷

It's been open for two weeks so I think I'll just merge 😄 this is reasonably low stakes as backups go

**Pull request merged**
